cmake/cpack/deb/packages/kodi-addon-dev.txt.in

```diff
@@ -16,7 +16,7 @@ PACKAGE_SECTION libdevel
 PACKAGE_PRIORITY optional
 PACKAGE_SHLIBDEPS
 PACKAGE_DEPENDS
-PACKAGE_RECOMMENDS @APP_NAME_LC@-audio-dev, @APP_NAME_LC@-inputstream-dev, @APP_NAME_LC@-pvr-dev, @APP_NAME_LC@-screensaver-dev, @APP_NAME_LC@-visualization-dev
+PACKAGE_RECOMMENDS
 PACKAGE_SUGGESTS
 PACKAGE_BREAKS
 PACKAGE_REPLACES
```

Comment:
I think `PACKAGE_REPLACES` should be populated with `@APP_NAME_LC@-audio-dev, @APP_NAME_LC@-inputstream-dev, @APP_NAME_LC@-pvr-dev, @APP_NAME_LC@-screensaver-dev, @APP_NAME_LC@-visualization-dev`

@wsnipex to confirm ^^